packages/git/src/index.ts

```
@@ -82,25 +84,28 @@ async function getChangedChangesetFilesSinceMaster(
   cwd: string,
   fullPath = false
 ): Promise<Array<string>> {
-  const ref = await getMasterRef(cwd);
-  // First we need to find the commit where we diverged from `ref` at using `git merge-base`
-  let cmd = await spawn("git", ["merge-base", ref, "HEAD"], { cwd });
-  // Now we can find which files we added
-  cmd = await spawn(
-    "git",
-    ["diff", "--name-only", "--diff-filter=d", "master"],
-    { cwd }
-  );
-
-  let tester = /.changeset\/[^/]+\.md$/;
-
-  const files = cmd.stdout
-    .toString()
-    .trim()
-    .split("\n")
-    .filter(file => tester.test(file));
-  if (!fullPath) return files;
-  return files.map(file => path.resolve(cwd, file));
+  try {
+    const ref = await getMasterRef(cwd);
+    // First we need to find the commit where we diverged from `ref` at using `git merge-base`
+    let cmd = await spawn("git", ["merge-base", ref, "HEAD"], { cwd });
+    // Now we can find which files we added
+    cmd = await spawn(
+      "git",
+      ["diff", "--name-only", "--diff-filter=d", "master"],
+      { cwd }
+    );
+
+    let tester = /.changeset\/[^/]+\.md$/;
+
+    const files = cmd.stdout
+      .toString()
+      .trim()
+      .split("\n")
+      .filter(file => tester.test(file));
+    if (!fullPath) return files;
+    return files.map(file => path.resolve(cwd, file));
+  } catch {}
```

Could we make this specific to the git errors so we don't silently catch errors from actual potential bugs.

---

Good pattern is:

```js
try {
  // some code at some time
} catch (e) {
  if (e.message.includes('this-specific-thing') {
    // handle the error
  } else {
    throw e
  }
}
```

message is one option, but there may be better things to interrogate.

---

One thing I like doing is this

```js
class GitError extends Error {}


try {
  // some code at some time
  throw new GitError('the message')
} catch (e) {
  if (e instanceof GitError) {
    // handle the error
  } else {
    throw e
  }
}
```